plugin_tests/dataone_register_test.py

```diff
@@ -0,0 +1,320 @@
+                   'resource_map_urn:uuid:1570fb91-69e8-4c1d-9b9a-292edb5820af']}]
+
+        data = []
+
+        expected_result = {'iso19139.xml': {'size': 14735}}
+
+        # In this case, the primary metadata object is the same as the metadata object
+        fileList = get_package_files(data, full_meta_data, full_meta_data)
+        assert fileList == expected_result
+
+
+    def test_get_package_files(self):
+        '''Test that the files in a package are getting correctly parsed. This is tested on
+        https://search.dataone.org/#view/urn:uuid:15403304-6eb8-4ede-8a56-332a3e92bef8
+        '''
+        from server.dataone_register import get_package_files
+
+        data = [{'identifier': 'urn:uuid:4eb73500-fa9b-46c2-a517-94c1a8b4afbb',
+                            'fileName': 'HumanFootprint.ipynb', 'formatId': 'text/plain',
+                            'formatType': 'DATA', 'size': 104524},
+                           {'identifier': 'urn:uuid:06544a24-aae8-4b80-be4b-bb03711d9fd0',
+                            'fileName': 'hfp_regions.csv', 'formatId': 'text/csv',
+                            'formatType': 'DATA', 'size': 1757}]
+
+        meta_data = [{'identifier': 'urn:uuid:15403304-6eb8-4ede-8a56-332a3e92bef8',
+                                  'fileName': 'HumanFootprint_SASAP.xml',
+                                  'formatId': 'eml://ecoinformatics.org/eml-2.1.1',
+                                  'formatType': 'METADATA', 'size': 18648,
+                                  'title': 'Global terrestrial Human Footprint maps for Alaska, 1993 and 2009,'
+                                    'with SASAP regional subsetting',
+                                  'documents': ['urn:uuid:15403304-6eb8-4ede-8a56-332a3e92bef8',
+                                                'urn:uuid:4eb73500-fa9b-46c2-a517-94c1a8b4afbb',
+                                                'urn:uuid:06544a24-aae8-4b80-be4b-bb03711d9fd0']}]
+
+        expected_result = {'HumanFootprint.ipynb': {'size': 104524}, 'hfp_regions.csv': {'size': 1757},
+                       'HumanFootprint_SASAP.xml': {'size': 18648}}
+
+        fileList = get_package_files(data, meta_data, meta_data)
+
+        assert fileList == expected_result
+
+
+    def test_check_multiple_maps_empty(self):
+        '''Test that we get an exception when no map was found.'''
+        from server.dataone_register import check_multiple_maps
+
+        with pytest.raises(RestException) as error:
+            metadata =set()
+            check_multiple_maps(metadata)
+
+    def test_get_package_list_nested(self):
+        # Test that we're getting all of the files in a nested package
+        from server.dataone_register import get_package_list
+
+        package = get_package_list("https://search.dataone.org/#view/urn:uuid:6f5533ab-6508-4ac7-82a3-1df88ed4580e")
+
+        # Metadata that should be returned
+        expected_result = {
```

Comment:
Can you export `expected_result` to a file and read it from disk to make this more readable?